secure-api-gateway-ob-uk-common-datamodel/src/main/java/com/forgerock/sapi/gateway/ob/uk/common/datamodel/converter/v4/common/FRErrorCodeConverter.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2020-2024 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.sapi.gateway.ob.uk.common.datamodel.converter.v4.common;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FRErrorCodeConverter {
+
+    private static final Map<String, String> v3tov4ErrorCodes;
+
+    static {
+        final Map<String, String> errorCodesTranslations = new HashMap<>();
+        errorCodesTranslations.put("UK.OBIE.Field.Expected", "U001");
```

Comment:
Could use OBStandardErrorCodes1 enum here but that would mean importing obie_datamodel... :thinking:

Reply:
Changed to use enum but had to update the OBStandardErrorCodes1 enum as it did not have all the values mentioned here:
https://github.com/OpenBankingUK/External_Internal_CodeSets/blob/main/OB_Internal_CodeSet.csv